src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -756,6 +753,27 @@ private void validateRules() {
           winnerMode.toString());
     }
 
+    if (!isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()
+        && !isMultiSeatAllowMultipleWinnersPerRoundEnabled()) {
```

Comment:
This condition is pretty confusing - can you add a comment on what we're checking for in this block?